actiontext/test/unit/model_test.rb

```diff
@@ -2,7 +2,7 @@
 
 require "test_helper"
 
-class ActionText::ModelTest < ActiveRecord::TestCase
+class ActionText::ModelTest < ActiveSupport::TestCase
```

Comment:
@dhh it seems that `ActiveRecord::TestCase` is not easy to use outside of the `activerecord` suite. I think this is preferably to putting the helpers directly in the single test and seems to follow the convention used in: `actionview/test/activerecord/relation_cache_test.rb`.

Open to other ideas if you have them. I wasn't ready to tackle more structural changes to `ActiveRecord::TestCase` based on my current understanding of the overall test suite.

Reply:
I tried fiddling around with this again today, but seems like the preference is to duplicate the query helpers vs making the existing AR::TestCase more accessible: https://github.com/rails/rails/pull/36031#discussion_r307984772